js/views/modals/editListing/EditListing.js

```diff
@@ -445,6 +447,9 @@ export default class extends BaseModal {
 
     this.$saveButton.addClass('disabled');
 
+    // one reason this is here since slots are not modified within validate
+    formData.listing.item.description = sanitizeHtml(formData.listing.item.description, htmlFilter);
```

Comment:
Let's remove this until we really need it. As of now, we're not allowing code-mode, so the user really won't be able to add any "bad" html.